### ISSUE TITLE
Bug running in kubernetes

### ISSUE THREAD

**Issue by wideklev** [marcells/node-build-monitor#96]
Hi!
Thanks for having a very good plugin. We run the build-monitor in Kubernetes keeping track of Gitlab builds.

Every once in awhile (Like once a week or so) the gitlab monitor runs blank and the following can be seen in the logs:
```
4:46:40 AM | 50 builds found....
4:47:10 AM | Check for builds...
/build-mon/app/services/GitLab.js:46
                    pipelines = pipelines.slice(0, self.config.numberOfPipelinesPerProject);
                                          ^
 TypeError: pipelines.slice is not a function
    at /build-mon/app/services/GitLab.js:46:43
    at Request._callback (/build-mon/app/services/GitLab.js:36:17)
    at Request.self.callback (/build-mon/node_modules/request/request.js:188:22)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:194:7)
    at Request.&lt;anonymous&gt; (/build-mon/node_modules/request/request.js:1171:10)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:191:7)
    at IncomingMessage.&lt;anonymous&gt; (/build-mon/node_modules/request/request.js:1091:12)
    at Object.onceWrapper (events.js:293:19)
error: Forever detected script exited with code: 1
error: Script restart attempt #26
Printing environment Variables...
    PORT = unset (Default: 3000)
    NODE_TLS_REJECT_UNAUTHORIZED = unset (Default: 1)
node-build-monitor is starting...
4:47:19 AM | Check for builds...
node-build-monitor 0.9.27 is listening on port 3000...
::ffff:10.231.55.108 - - [04/Jan/2018:14:28:27 +0000] "GET /favicon.ico HTTP/1.1" 404 150 "http://example.com/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.84 Safari/537.36"
```

The pod is not restarted or considered corrupt so it stays in this state until the pod is recreated.
If there is any other information I can provide, please let me know.

**Comment by marcells** [comment]
Looks strange. Maybe some kind of a network issue.

Could you configure the container to restart automatically, if it fails as a quick solution?

Otherwise maybe some retry-mechanism is needed.

**Comment by marcells** [comment]
This bug was fixed in release v0.9.32. You are using v0.9.27. Please update to the latest version.

If there are still any problems, please reopen this issue.
